eahub/config/settings.py

```diff
@@ -317,7 +317,7 @@ class DjangoEnv(Enum):
         "app": "sites",
         "label": "Website administration",
         "models": [
-            {"model": "auth.User", "label": "User accounts"},
+            {"model": "base.User", "label": "User accounts"},
```

Comment:
What's the value of having user accounts and user account email addresses? Wouldn't the first suffice?

Reply:
I don't see value in it, but the multiple email per user feature already exists and it can't be disabled with a simple patch now